package.json

```diff
@@ -34,9 +34,10 @@
   "scripts": {
     "precommit": "./pre-commit.sh",
     "start": "node --trace-warnings ./server/bin/www",
-    "dev": "yarn run heroku-prebuild && yarn run start",
+    "dev": "yarn run heroku-prebuild && yarn run migration && yarn run start",
```

Comment:
Does this only result in migration being run locally? Or do we use `dev` for the staging/prod deploys?

Reply:
Good point, Heroku uses `start`

Reply:
Updated